SingularityUI/app/components/taskDetail/ShellCommands.jsx

```diff
@@ -85,45 +114,43 @@ export default class ShellCommands extends React.Component {
       </div>
     );
 
-    const history = !!this.props.task.shellCommandHistory.length && (
+    const history = !!this.props.shellCommandHistory.length && (
       <div>
         <h3>Command History</h3>
           <SimpleTable
             emptyMessage="No commands run"
-            entries={this.props.task.shellCommandHistory}
+            entries={this.props.shellCommandHistory}
             perPage={5}
             first={true}
             last={true}
             headers={['Timestamp', 'Command', 'User', 'Status', 'Message', '']}
             renderTableRow={(data, index) => {
-              let updates = _.sortBy(data.shellUpdates, 'timestamp');
-              let withFilename = _.find(updates, (u) => u.outputFilename);
-              let filename = withFilename ? withFilename.outputFilename : null;
+              const updates = _.sortBy(data.shellUpdates, 'timestamp');
+              const withFilename = _.find(updates, (update) => update.outputFilename);
+              const filename = withFilename && withFilename.outputFilename;
               return (
                 <tr key={index}>
                   <td>{Utils.absoluteTimestamp(data.shellRequest.timestamp)}</td>
                   <td><code>{data.shellRequest.shellCommand.name}</code></td>
                   <td>{data.shellRequest.user}</td>
-                  <td>{updates.length ? _.last(updates).updateType : null}</td>
+                  <td>{updates.length && _.last(updates).updateType}</td>
                   <td>
                     <ul>
-                      {updates.map((u) => {
-                        return <li key={u.timestamp}>{Utils.absoluteTimestamp(u.timestamp)}: {u.message}</li>;
+                      {updates.map((update) => {
+                        return <li key={update.timestamp}>{Utils.absoluteTimestamp(update.timestamp)}: {update.message}</li>;
                       })}
                     </ul>
                   </td>
                   <td className="actions-column">
-                    {filename ? (
+                    {filename && (
                       <Link prop={{
                         url: `${config.appRoot}/task/${data.shellRequest.taskId.id}/tail/${data.shellRequest.taskId.id}/${filename}`,
                         text: '···',
                         overlayTrigger: true,
                         overlayId: filename,
                         overlayTriggerPlacement: 'left',
-                        overlayToolTipContent: 'View output file'
-                      }}
-                      />
-                    ) : null}
+                        overlayToolTipContent: 'View output file'}} />
```

Comment:
Note - this is a linting error in my Sublime eslint plugin. But it is not an error when running the eslint command on the command line.